src/Aspire.Hosting.Azure.AppConfiguration/README.md

```diff
@@ -16,6 +16,27 @@ In your AppHost project, install the .NET Aspire Azure App Configuration Hosting
 dotnet add package Aspire.Hosting.Azure.AppConfiguration
 ```
 
+## Configure Azure Provisioning for local development
+
+Adding Azure resources to the .NET Aspire application model will automatically enable automated provisioning
+for Azure resources so that you don't need to configure them manually. Provisioning requires a number of settings
```

Comment:
```suggestion
Adding Azure resources to the .NET Aspire application model will automatically enable development-time provisioning
for Azure resources so that you don't need to configure them manually. Provisioning requires a number of settings
```

1. not sure we need to say "automatically enable automated"
2. We should be clear this is development-time only